pkgs/development/interpreters/python/hooks/wheel-deps-versions-hook.sh

```diff
@@ -0,0 +1,82 @@
+# shellcheck shell=bash
+
+# Setup hook that modifies Python dependencies versions.
+#
+# Example usage in a derivation:
+#
+#   { …, pythonPackages, … }:
+#
+#   pythonPackages.buildPythonPackage {
+#     …
+#     nativeBuildInputs = [ pythonPackages.wheelDepsVersionsHook ];
+#
+#     # This will relax the dependency restrictions
+#     # e.g.: abc>1,<=2 -> abc
+#     relaxPythonDeps = [ "abc" ];
```

Comment:
`wheelRelaxDeps`

Reply:
Makes sense for consistency. If I don't change this hook to `pythonSomethingSomethingHook` like suggested in the other comment I will make this change 👍 .